src/test/scala/org/zalando/kanadi/OAuthFailedSpec.scala

```diff
@@ -27,12 +26,22 @@ class OAuthFailedSpec extends Specification with FutureMatchers with Config {
     Subscriptions(nakadiUri, failingauthTokenProvider)
   val eventsClient = Events(nakadiUri, failingauthTokenProvider)
 
-  override def is: SpecStructure = s2"""
-    Call to subscriptions list should fail with invalid token   $oAuthCallSubscriptions
-    Call to publishEvents should fail with invalid token        $oAuthPublishEvents
-  """
-
-  def oAuthCallSubscriptions = Skipped("No way for current Nakadi docker image to detect \"wrong\" tokens")
-
-  def oAuthPublishEvents = Skipped("No way for current Nakadi docker image to detect \"wrong\" tokens")
+  property("Call to subscriptions list should fail with invalid token") { implicit testData: TestData =>
```

Comment:
Scalatest's ignore properly supports having code contained inside which doesn't run unlike with specs2 so I have gone ahead and undid the removal of the test code.

Even though it doesn't work with the current Nakadi docker image, the code illustrates what the expected behaviour is which is useful